MicroProfile-Fault-Tolerance/tck-runner/pom.xml

```diff
@@ -96,14 +92,42 @@
                 <artifactId>maven-failsafe-plugin</artifactId>
                 <configuration>
                     <suiteXmlFiles>
-                        <suiteXmlFile>${basedir}/src/test/resources/stable-no-ext-libs-tck-suite.xml</suiteXmlFile>
+                        <suiteXmlFile>${basedir}/src/test/resources/${microprofile.fault-tolerance.tck.suite}</suiteXmlFile>
                     </suiteXmlFiles>
                 </configuration>
             </plugin>
         </plugins>
     </build>
 
     <profiles>
+        <profile>
+            <id>mp-ft-1.1</id>
+            <activation>
+                <property>
+                    <name>payara.version</name>
+                    <value>/5\.18.+|5\.191.*/</value>
```

Comment:
Misses Payara4

Reply:
I left out 4 since 4 does not even have  MP FT support and I was assuming we would not have or run this test on 4. Is this wrong?

Reply:
Does payara 4 support mp-ft?

Reply:
4 currently supports FT 1.1.1.

Reply:
I included `4.*` as well.